SparkPost/2web2ui
# EO-752 Add recommendations for Complaint Rate by cohort

### What Changed
 - Added recommendations content for complaint rate by cohort. Values from here: https://docs.google.com/spreadsheets/d/1AOcX1eQ6flweFV0O3vFTaYyG0h_QC5e7LHLvS7Mjdsg/edit#gid=92043718.
The recommendation text is a function because 3 warnings can be merged together. 
- Added component that generates the helper messages. 
- Added test.

### How To Test
 - [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data ( production account 108 works best because it has actual data)
- Visit v2 Engagement Recency dashboard
- Click on one of the accounts
- Click on `Complaint Rate` tab.
- Check that all recommendations are the correct ones. 
### To Do
- [ ] Maybe change some tests from snapshots to explicit assertions
